nbs/15_gsa_gls/30-real_phenotypes/05-group_gwas_by_n_variants.ipynb

```diff
@@ -0,0 +1,1599 @@
+{
```

Comment:
<p>why is this section commented out?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/phenoplier/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>

Reply:
For future reference. I added a description.